src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/service/NomisApiService.kt

```diff
@@ -19,6 +19,7 @@ class NomisApiService(@Qualifier("nomisApiWebClient") private val webClient: Web
     visitTypes: List<String>,
     fromDateTime: LocalDateTime?,
     toDateTime: LocalDateTime?,
+    ignoreMissingRoom: Boolean,
```

Comment:
would it be better as nullable? It will default to false in the api if parameter not provided

Reply:
I did consider that, but went for the default of false which functionally will work the same